android/Dockerfile

```diff
@@ -23,12 +23,15 @@ RUN yes | sudo $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --licenses
 RUN sudo $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager tools platform-tools
 ENV PATH=$ANDROID_SDK_ROOT/cmdline-tools/latest/bin:$ANDROID_SDK_ROOT/platform-tools:$PATH
 
-# Gradle
-RUN sudo wget https://services.gradle.org/distributions/gradle-8.9-bin.zip -O /opt/archives/gradle-8.9-bin.zip
-RUN sudo unzip /opt/archives/gradle-8.9-bin.zip -d /opt
-ENV GRADLE_HOME=/opt/gradle-8.9
-ENV PATH=$GRADLE_HOME/bin:$PATH
```

Comment:
We always use gradle from the wrapper - system-level gradle won't help us, so I decided to cut it out of the build